src/components/immersiveCaption.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, ReactElement } from 'react';
+import { Item } from 'item';
+import { MainMedia, MainMediaKind } from 'headerMedia';
+import { Option, OptionKind } from '@guardian/types/option';
+import { renderCaption } from 'components/figCaption'
+import { Format } from "format";
```

Comment:
```suggestion
import { Format } from '@guardian/types/Format';
```

Reply:
This looks like it's an error in the CI

Reply:
had trouble pushing the new changes

Reply:
Apparently there is a problem with the format:
TS7016: Could not find a declaration file for module 'format'. '/home/runner/work/apps-rendering/apps-rendering/node_modules/format/format.js' implicitly has an 'any' type.
Is this due to the format file change Jamie did?